segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -623,6 +625,13 @@ private void recordStatForTransaction(SegmentProperties sourceInfo, String targe
                 if (statsRecorder != null) {
                     statsRecorder.merge(targetSegmentName, len, numOfEvents, creationTime);
                 }
+
+                //if source segment is a transaction, add its length and event count onto metrics of target segment
```

Comment:
Please, according to our guidelines, comments start by space and capitals.

Reply:
fixed